src/test/resources/archives/caseclaim/suite.xml

```diff
@@ -408,6 +307,18 @@
             <sort ref="id"/>
           </itemset>
         </prompt>
+        <prompt key="age">
```

Comment:
this is the only meaningful change in test file. Rest is - 
- earlier caseclaim folder was renamed to case_claim_post_in_query
- caseclaim.ccz was unarchived to caseclaim

Somehow github is screwing this diff.